src/main/java/de/dataelementhub/model/handler/export/ExportMembersHandler.java

```diff
@@ -0,0 +1,153 @@
+package de.dataelementhub.model.handler.export;
+
+import de.dataelementhub.dal.jooq.enums.ElementType;
+import de.dataelementhub.model.dto.element.section.Identification;
+import de.dataelementhub.model.dto.element.section.Member;
+import de.dataelementhub.model.handler.element.NamespaceHandler;
+import de.dataelementhub.model.handler.element.section.IdentificationHandler;
+import de.dataelementhub.model.handler.element.section.MemberHandler;
+import de.dataelementhub.model.handler.element.section.ValueDomainHandler;
+import de.dataelementhub.model.handler.element.section.validation.PermittedValuesHandler;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.jooq.DSLContext;
+
+/**
+ * Handle export members.
+ */
+public class ExportMembersHandler {
+
+  /**
+   * Inside the function:
+   *  1. Identify the elementType for every exportMember
+   *  2. Add the related exportMembers according to the elementType
+   *  3. Return the extended exportMembers list
+   *
+   * @param ctx configures jOOQ's behaviour when executing queries
+   * @param userId represents the current user
+   * @param elementUrns represent the super urns that have to be extended
+   * @return a list of all exportMembers
+   */
+  public static List<Member> getAllExportMembers(
+      DSLContext ctx, int userId, List<String> elementUrns) {
+    List<Member> exportMembers = new ArrayList<>();
+    for (String elementUrn : elementUrns) {
+      ElementType elementType =
+          IdentificationHandler.getScopedIdentifier(ctx, elementUrn).getElementType();
+      switch (elementType) {
+        case NAMESPACE: // All namespace members should be exported too.
+          String[] parts = elementUrn.split(":");
```

Comment:
I think you could use IdentificationHandler.getNamespaceIdentifierFromUrn() here, since you only use that part anyways.